src/javascript/app/pages/user/account/settings/personal_details.js

```diff
@@ -238,11 +238,13 @@ const PersonalDetails = (() => {
                     return;
                 }
                 getDetailsResponse(get_settings);
+                if (additionalCheck(get_settings)) {
+                    showFormMessage(localize('Your settings have been updated successfully.'), !is_error);
```

Comment:
this is already inside the `!is_error` if condition, so you can simply send `1` or `true` instead